ironfish/src/primitives/target.test.ts

```diff
@@ -112,7 +112,7 @@ describe('Calculate target', () => {
   it('dencreases difficulty if a new block is coming in after the target range time', () => {
     const now = new Date()
 
-    // for any time more than 65 seconds after the last block, difficulty should decrease by previous block's difficulty / BigInt(2048) / n
+    // for any time more than 65 seconds after the last block, difficulty should decrease by previous block's difficulty / BigInt(2048) / bucket
```

Comment:
`previous block's difficulty / BigInt(2048) / bucket`

that's not the algorithm though.

Reply:
we're never dividing by bucket